src/sensors.rs

```diff
@@ -88,10 +88,22 @@ pub struct TemperatureSensor {
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 pub struct Sensors {
-    #[serde(skip_serializing_if = "Vec::is_empty")]
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
```

Comment:
If the field is missing when deserializing, it will be set to the `Default` (empty for Vec). See https://serde.rs/attributes.html#field-attributes